analytic_operating_unit_access_all/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Access all OUs' Analytics",
-    "version": "15.0.1.0.0",
+    "version": "17.0.1.0.0",
```

Comment:
@traitrantrobz Migrated codebase should be in the migration commit, not pre-commit commit